app/src/main/java/io/pumpkinz/pumpkinreader/model/Item.java

```diff
@@ -0,0 +1,129 @@
+package io.pumpkinz.pumpkinreader.model;
+
+import java.util.Date;
+
+
+public abstract class Item {
```

Comment:
ItemPOJO is a class to contain the return value of Retrofit API. Retrofit will automatically parse the JSON, instantiate the class and map its JSON values into the class' fields (in this case, the ItemPOJO class).